src/QMCWaveFunctions/ConstantSPOSet.h

```diff
@@ -0,0 +1,133 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
```

Comment:
Mock classes should go in the testing directories not module roots.  See `src/Estimators/tests/FakeOperatorEstimator.h`  Generally they should not be header only. Making the tests build slower benefits nobody.  The targets they are built as part of shouldn't be linked into the application code.

If the class is with testing code and only linked to a test program or sub target of a test program it may not be necessary to call it Fake... or Mock...

Reply:
My intended use case here is for Estimator/Hamiltonian evaluation, secondarily as a test of WaveFunctionComponents, so they would be linked to stuff in both of those directories.  Specifically, want to get rid of some of the SPOSet gymnastics done in things like test_soecp.cpp (and soon to be in magnetization density).  What do you recommend in this case?  Same location and naming convention?

Reply:
Right now we do share tests classes across modules, all the minimal pools for instance.  At the moment the easier thing to do in that case is to leave the test class header only. Include by full path to the file from the project root if crossing modules.
ex:
```
#include "Particle/tests/MinimalParticlePool.h"
```

Reply:
The only one that seems to make a target that gets linked by other tests is container_testing from the Containers module. But I think its overkill for this class.